src/cdk/drag-drop/drag-events.ts

```diff
@@ -71,3 +71,15 @@ export interface CdkDragMove<T = any> {
    */
   delta: {x: -1 | 0 | 1, y: -1 | 0 | 1};
 }
+
+/** Event emitted when the user swaps the position of two drag items. */
+export interface CdkDragSort<T = any, I = T> {
```

Comment:
Can you call this CdkDragSortEvent? I forgot to mention this, but for v8 we want to make all of our emitted event objects consistently named with Event at the end.

Reply:
Sure, but it'll be the odd one out since none of the other events have it.